MREUnityRuntime/MREUnityRuntimeLib/Core/ColliderGeometry.cs

```diff
@@ -60,7 +61,7 @@ private void Patch(SphereCollider collider)
 
             if (Radius != null)
             {
-                collider.radius = Radius.Value;
+                collider.radius = Mathf.Abs(Radius.Value);
```

Comment:
All of these Mathf.Abs calls are unnecessary.  Unity already does this for these values under the hood.  This change is redundant and is why it wasn't there originally.